CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# HBASE Changelog
```

Comment:
this file needs a ASF license header. if you run release doc maker with the `--license` option it should do it for you.